src/cdk/v2/destinations/sprig/procWorkflow.yaml

```diff
@@ -19,6 +19,12 @@ steps:
       $.assert(messageType in {{$.EventType.([.IDENTIFY, .TRACK])}}, "message type " + messageType + " is not supported");
       $.assertConfig(.destination.Config.apiKey, "API Key is not present. Aborting");
 
+  - name: validateIdentifyPayload
```

Comment:
[optional]you can also say common validation and the in track only validate .message.event